ui/pages/onboarding-flow/add-network-modal/index.js

```diff
@@ -34,6 +34,7 @@ export default function AddNetworkModal() {
       </Box>
       <NetworksForm
         addNewNetwork
+        restrictHeight
```

Comment:
Optional: Sorry for late review, but i think we can use name like `isHeightRestricted` to represent better to be a boolean. Totally optional, I just found out it is useful to pick up when we don't have rich document around props.